Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM registry.access.redhat.com/ubi9/ubi-minimal
 
 COPY --from=builder workspace/ocs-operator /usr/local/bin/ocs-operator
 COPY --from=builder workspace/provider-api /usr/local/bin/provider-api
-COPY --from=builder workspace/onboarding-secret-generator /usr/local/bin/onboarding-secret-generator
+COPY --from=builder workspace/onboarding-validation-keypairs-generator /usr/local/bin/onboarding-validation-keypairs-generator
```

Comment:
I believe we agreed on `onboarding-validation-keys-generator` and not `onboarding-validation-keypairs-generator`

Reply:
see https://github.com/red-hat-storage/ocs-operator/pull/2452/files#r1510218145